src/main/java/ch/heigvd/dai/commands/Root.java

```diff
@@ -0,0 +1,66 @@
+package ch.heigvd.dai.commands;
```

Comment:
Missing header / copyright

Reply:
This is already being done in [this branch](https://github.com/lentidas/DAI-2425-PW1/tree/docs/add_readme_documentation).